core/block.js

```diff
@@ -258,6 +271,23 @@ Blockly.Block.prototype.dispose = function(healStack) {
   }
 };
 
+/**
+ * Call initModel on all fields on the block.
+ * May be called more than once.
+ * Either initModel or initSvg must be called after creating a block and before
+ * first interaction with it.
```

Comment:
Nit: ...before _the_ first...
What defines an interaction? Strictly UI, or are some API calls also prohibited before calling init?

Reply:
Updated comment: "Interactions include UI actions (e.g. clicking and dragging) and firing events (e.g. create, delete, and change)."